From 6ad905f6616fd401d33e597187393f226d9cdb78 Mon Sep 17 00:00:00 2001
From: De Huo <De.Huo@windriver.com>
Date: Fri, 9 Aug 2019 14:38:39 +0800
Subject: [PATCH] Replace rules accounts_password_pam_* with
 cracklib_accounts_password_pam_*

    cracklib_accounts_password_pam_ucredit
    cracklib_accounts_password_pam_lcredit
    cracklib_accounts_password_pam_dcredit
    cracklib_accounts_password_pam_difok
    cracklib_accounts_password_pam_minclass
    cracklib_accounts_password_pam_maxrepeat
    cracklib_accounts_password_pam_ocredit
Use above rules to replace below rules.
    accounts_password_pam_ucredit
    accounts_password_pam_lcredit
    accounts_password_pam_dcredit
    accounts_password_pam_difok
    accounts_password_pam_minclass
    accounts_password_pam_maxrepeat
    accounts_password_pam_ocredit


Upstream-Status: Pending

Signed-off-by: De Huo <De.Huo@windriver.com>
---
 wrlinux1019/profiles/draft_stig_wrlinux_disa.profile | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile b/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile
index c2a6f3f..90c6cf0 100644
--- a/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile
+++ b/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile
@@ -196,12 +196,12 @@ selections:
     - audit_rules_dac_modification_fremovexattr
     - audit_rules_dac_modification_lremovexattr
     - cracklib_accounts_password_pam_retry
-    - accounts_password_pam_ucredit
-    - accounts_password_pam_lcredit
-    - accounts_password_pam_dcredit
-    - accounts_password_pam_difok
-    - accounts_password_pam_minclass
-    - accounts_password_pam_maxrepeat
+    - cracklib_accounts_password_pam_ucredit
+    - cracklib_accounts_password_pam_lcredit
+    - cracklib_accounts_password_pam_dcredit
+    - cracklib_accounts_password_pam_difok
+    - cracklib_accounts_password_pam_minclass
+    - cracklib_accounts_password_pam_maxrepeat
     - accounts_password_pam_maxclassrepeat
     - set_password_hashing_algorithm_systemauth
     - accounts_password_pam_minlen
@@ -221,7 +221,7 @@ selections:
     - sssd_ldap_start_tls
     - sssd_ldap_configure_tls_ca_dir
     - sssd_ldap_configure_tls_ca
-    - accounts_password_pam_ocredit
+    - cracklib_accounts_password_pam_ocredit
     - ensure_gpgcheck_local_packages
     - grub2_no_removeable_media
     - auditd_data_retention_admin_space_left_action
-- 
1.9.1

